File: Baekjoon/C++/15552. 빠른 A+B/solution.cpp
// https://www.acmicpc.net/problem/15552

#include <iostream>

using namespace std;

int main(int argc, char const *argv[])
{
    int n, a, b;
    scanf("%d", &n);
    for (int i = 0; i < n; i++)
    {
        scanf("%d %d", &a, &b);
        printf("%d\n", a + b);
    }
}

File: Baekjoon/Go/15552. 빠른 A+B/solution.go
// https://www.acmicpc.net/problem/15552

package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	writer := bufio.NewWriter(os.Stdout)
	var n int
	fmt.Scanln(&n)

	var a, b int
	for i := 0; i < n; i++ {

		fmt.Fscanln(reader, &a, &b)
		fmt.Fprintln(writer, a + b)
	}
	writer.Flush()	// 빠른 입출력 사용시 반드시 Flush()를 사용해야 함
}

